pkg/scheduler/apis/config/validation/validation_pluginargs.go

```diff
@@ -182,81 +188,90 @@ func validateConstraintNotRepeat(path *field.Path, constraints []v1.TopologySpre
 
 // ValidateRequestedToCapacityRatioArgs validates that RequestedToCapacityRatioArgs are correct.
 func ValidateRequestedToCapacityRatioArgs(args config.RequestedToCapacityRatioArgs) error {
-	if err := validateFunctionShape(args.Shape); err != nil {
-		return err
-	}
-	if err := validateResourcesNoMax(args.Resources); err != nil {
-		return err
-	}
-	return nil
+	var allErrs field.ErrorList
+	allErrs = append(allErrs, validateFunctionShape(args.Shape, field.NewPath("shape"))...)
+	allErrs = append(allErrs, validateResourcesNoMax(args.Resources, field.NewPath("resources"))...)
+	return allErrs.ToAggregate()
 }
 
-func validateFunctionShape(shape []config.UtilizationShapePoint) error {
+func validateFunctionShape(shape []config.UtilizationShapePoint, p *field.Path) field.ErrorList {
 	const (
 		minUtilization = 0
 		maxUtilization = 100
 		minScore       = 0
 		maxScore       = int32(config.MaxCustomPriorityScore)
 	)
 
+	var allErrs field.ErrorList
+
 	if len(shape) == 0 {
-		return fmt.Errorf("at least one point must be specified")
+		allErrs = append(allErrs, field.Required(p, "at least one point must be specified"))
+		return allErrs
 	}
 
 	for i := 1; i < len(shape); i++ {
 		if shape[i-1].Utilization >= shape[i].Utilization {
-			return fmt.Errorf("utilization values must be sorted. Utilization[%d]==%d >= Utilization[%d]==%d", i-1, shape[i-1].Utilization, i, shape[i].Utilization)
+			msg := fmt.Sprintf("utilization values must be sorted. Should < %s==%d", p.Index(i).Child("utilization").String(), shape[i].Utilization)
+			allErrs = append(allErrs, field.Invalid(p.Index(i-1).Child("utilization"), shape[i-1].Utilization, msg))
 		}
 	}
 
 	for i, point := range shape {
+		utilizationPath := p.Index(i).Child("utilization")
 		if point.Utilization < minUtilization {
-			return fmt.Errorf("utilization values must not be less than %d. Utilization[%d]==%d", minUtilization, i, point.Utilization)
+			allErrs = append(allErrs, field.Invalid(utilizationPath, point.Utilization, fmt.Sprintf("must not be less than %d", minUtilization)))
 		}
 		if point.Utilization > maxUtilization {
-			return fmt.Errorf("utilization values must not be greater than %d. Utilization[%d]==%d", maxUtilization, i, point.Utilization)
+			allErrs = append(allErrs, field.Invalid(utilizationPath, point.Utilization, fmt.Sprintf("must not be greater than %d", maxUtilization)))
 		}
+
+		scorePath := p.Index(i).Child("score")
 		if point.Score < minScore {
-			return fmt.Errorf("score values must not be less than %d. Score[%d]==%d", minScore, i, point.Score)
+			allErrs = append(allErrs, field.Invalid(scorePath, point.Score, fmt.Sprintf("must not be less than %d", minScore)))
```

Comment:
For consistency with other messages, make the detail:

`not in the range [x, y]`